### PR TITLE
make offline deploy faster

### PR DESCRIPTION
**Pull request by snopoke**
decided to just make some of these changes since I wanted to do a deploy anyways:

* use ansible user
* re-use remote offline code root
* rsync files to one machine from local and then rsync from that machine to all the others

### CODE REVIEW

**Review by calellowitz — APPROVED** [review]
👍 this looks great and will be super useful if we get stuck doing more offline deploys.
Thanks for doing this.

File: fab/fabfile.py [hunk 1]
```diff
@@ -186,6 +189,9 @@ def icds():
     env.inventory = os.path.join(PROJECT_ROOT, 'inventory', 'icds')
     load_env('icds')
     env.force = True  # don't worry about kafka checkpoints on icds
+    # Force ansible user and prompt for password
+    env.user = 'ansible'
+    env.sudo_password = getpass('Enter the password for then ansbile user: ')
```
Comment (benrudolph) [inline fab/fabfile.py]: should we only do this if we're doing an offline deploy?

File: fab/operations/release.py [hunk 2]
```diff
@@ -58,6 +55,44 @@ def create_offline_dir():
...
+            os.path.join(OFFLINE_STAGING_DIR, 'commcare-hq', sync_item),
+            delete=True,
+        )
+    rsync_project(
+        env.offline_code_dir,
+        os.path.join(OFFLINE_STAGING_DIR, 'formplayer.jar'),
+    )
+
+
+def sync_offline_from_control():
```
Comment (benrudolph) [inline fab/operations/release.py:77]: sweet

**Review by benrudolph — APPROVED** [review]
nicely done 👏

**Comment by benrudolph** [comment]
@snopoke just an fyi adding the control machine to the inventory files causes ansible to fail on the become user. it doesn't stop us from executing tasks though
```
fatal: [control.internal-va.commcarehq.org] => Incorrect become password
```

**Comment by snopoke** [comment]
is that because the control machine doesn't have the `cchq` user?

**Comment by benrudolph** [comment]
i believe so

**Comment by sravfeyn** [comment]
Hey @snopoke should we revert some of these changes, now that we don't need them anymore? For e.g. ansible password prompt when doing icds deploy

**Comment by benrudolph** [comment]
i think we should skip the prompt and ansible user if the deploy isn't offline. otherwise i think these changes are good, you never know when we may have to deploy offline again...

**Comment by sravfeyn** [comment]
Yeah definitely, meant to revert only any hacks like the example I gave.

**Pull request merged**